lib/src/util/io.dart

```diff
@@ -18,6 +18,23 @@ const _newline = 0xA;
 /// The ASCII code for a carriage return character.
 const _carriageReturn = 0xD;
 
+/// The default line length for output when there isn't a terminal attached to
+/// stdout.
+const _defaultLineLength = 200;
+
+/// The maximum line length for output.
+final int lineLength = () {
+  try {
+    return stdout.terminalColumns;
+  } on UnsupportedError catch (_) {
```

Comment:
[nit] omit the `catch` if we are ignoring the caught value.

Reply:
Done.